spec/ics-1-ics-standard/README.md

```diff
@@ -0,0 +1,151 @@
+---
+ics: 1
+title: ICS Specification Standard
+stage: draft
+category: meta
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-12
+modified: 2019-03-04
+---
+
+## What is an ICS?
+
+An inter-chain standard (ICS) is a design document describing a particular protocol,
+standard, or feature expected to be of use to the Cosmos ecosystem.
+An ICS should list the desiderata of the standard, explain the design rationale, and
+provide a concise but comprehensive technical specification. The primary ICS author
+is responsible for pushing the proposal through the standardization process, soliciting
+input and support from the community, and communicating with relevant stakeholders to
+ensure (social) consensus.
+
+The inter-chain standardization process should be the primary vehicle for proposing
+ecosystem-wide protocols, changes, and features, and ICS documents should persist after
+consensus as a record of design decisions and an information repository for future implementers.
+
+Inter-chain standards should *not* be used for proposing changes to a particular blockchain
+(such as the Cosmos Hub), specifying implementation particulars (such as language-specific data structures),
+or debating governance proposals on existing Cosmos blockchains (although it is possible
+that individual blockchains in the Cosmos ecosystem may utilize their governance processes
+to approve or reject inter-chain standards).
+
+## Components
+
+An ICS consists of a header, synopsis, specification, history log, and copyright notice. All top-level sections are required.
+
+### Header
+
+An ICS header contains metadata relevant to the ICS.
+
+#### Required fields
+
+`ics: #` - ICS number (assigned sequentially)
+
+`title` - ICS title (keep it short & sweet)
+
+`stage` - Current ICS stage, one of the following:
+- `proposal` - A standard in the "proposal" stage
+- `draft` - A standard in the "draft" stage
+- `candidate` - A standard in the "candidate" stage
+- `finalized` - A standard in the "finalized" stage
+
+See [README.md](../../README.md) for a description of the ICS acceptance stages.
+
+`category` - ICS category, one of the following:
+- `meta` - A standard about the ICS process
+- `ibc`  - A standard about the inter-blockchain communication system
+- `util` - A standard about utility features, e.g. message signing
```

Comment:
Is there a possible better descriptor than `util`? `common` maybe, although that is not much better.

Reply:
`aux` (for `auxiliary`) or `misc` (for `miscellaneous`)?

Reply:
I like misc/miscellaneous 👍

Reply:
Updated.